trade/src/main/proto/trade.proto

```
@@ -19,6 +19,7 @@ syntax = "proto3";
 
 package trade;
 
+import "common.proto";
```

Buf (1.54.0)

import "common.proto": file does not exist

(COMPILE)